packages/block-editor/src/hooks/metadata.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
```

Comment:
This import from lodash is now restricted and is causing a linting error.

Reply:
Yeh it's unnecessary. I'll sort that. Thanks!

Reply:
This possibly still needs to be addressed? Not sure.

Reply:
Thanks for flagging. I missed it. 

✅